changelog/@unreleased/pr-5759.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: improvement
+improvement:
+  description: The AtlasBackupService will now verify the consistency of backups by
```

Comment:
I guess we also actually store the state and the other necessary data

Reply:
well now we do - we didn't when I wrote this changelog :)